### PR TITLE
webui: parameterize ports for ssh, cockpit connection and http server

### PR DESCRIPTION
**Pull request by KKoukiou**
Previously no parallel test runs were possible, since there was port collision on the host.
This commit makes it possible for local test runs or CI to run in
paralle.

### CODE REVIEW

File: ui/webui/test/run-test [hunk 1]
```diff
@@ -4,13 +4,36 @@ set -e
...
+ssh_port=22000
+until echo '' | nc 127.0.0.2 $ssh_port 2>&1 >/dev/null | grep -q 'Connection refused'; do
+    ssh_port=$((ssh_port+1))
+done
+
+# determine cockpit forward port
+cockpit_port=9091
+until echo '' | nc 127.0.0.2 $cockpit_port 2>&1 >/dev/null | grep -q 'Connection refused'; do
+    cockpit_port=$((cockpit_port+1))
+done
```
Comment (jkonecny12) [inline ui/webui/test/run-test:31]: I wonder if there is no way to tell - just give me a free port?

**Review by jkonecny12 — APPROVED** [review]
Looks good to me. Just one loud thinking.

Also you are missing `l` in the commit message last word.

File: ui/webui/test/run-test (hunk 1 above)
Comment (KKoukiou) [reply ui/webui/test/run-test:31]: That would be really cool - like - give me a free port in that range - but I am not familiar with it.

**Comment by KKoukiou** [comment]
> Looks good to me. Just one loud thinking.
> 
> Also you are missing `l` in the commit message last word.

Unfortunately this PR is making webui tests fail, and I don't know yet why, I will need to investigate!

**Comment by KKoukiou** [comment]
Opened the PR to the original repo.

**Pull request closed without merging**